# Internal, will not be submitted upstream.
# Solaris path to gmp headers.

--- php-5.6.3/ext/gmp/config.m4	Wed Nov 12 05:52:21 2014
+++ php-5.6.3/ext/gmp/config.m4	Fri Mar  6 13:46:41 2015
@@ -9,6 +9,7 @@
 
   for i in $PHP_GMP /usr/local /usr; do
     test -f $i/include/gmp.h && GMP_DIR=$i && break
+    test -f $i/include/gmp/gmp.h && GMP_DIR=$i && break
   done
 
   if test -z "$GMP_DIR"; then
@@ -29,6 +30,7 @@
 
   PHP_ADD_LIBRARY_WITH_PATH(gmp, $GMP_DIR/$PHP_LIBDIR, GMP_SHARED_LIBADD)
   PHP_ADD_INCLUDE($GMP_DIR/include)
+  PHP_ADD_INCLUDE($GMP_DIR/include/gmp)
 
   PHP_NEW_EXTENSION(gmp, gmp.c, $ext_shared)
   PHP_SUBST(GMP_SHARED_LIBADD)
